zio/zjsonio/stream.go

```diff
@@ -44,6 +45,58 @@ func (s *Stream) Transform(r *zng.Record) (*Record, error) {
 	}, nil
 }
 
+func encodeUnion(typ *zng.TypeUnion, v []byte) (interface{}, error) {
+	// encode nil val as JSON null since
+	// zng.Escape() returns "" for nil
+	var fld interface{}
+	if v == nil {
+		return fld, nil
+	}
+
+	// We start out with a slice that contains nothing instead of nil
+	// so that an empty containers encode to JSON empty array [].
```

Comment:
This comment doesn't make sense.

Reply:
spurious cut'n'paste from elsewhere, removed.